UMI3D-SDK/Assets/UMI3D SDK/ClientDevelopmentKit/Core/Runtime/Loaders/IUMI3DLoadingParameters.cs

```diff
@@ -31,6 +31,8 @@ public interface IUMI3DLoadingParameters : IUMI3DAbstractLoadingParameters
         Material skyboxMaterial { get; }
         KHR_lights_punctualLoader khr_lights_punctualLoader { get; }
 
+        bool IsBrowserImmersive { get; }
```

Comment:
"IsImmervise" is by legacy a term defining if the device is in Xr or Vr. Please choose an other name.

Reply:
Reverted changes to previous term in this class.

Added HasImmersiveDisplay boolean to know whether a bowser uses an immersive display.